tools/ports/sdl2.py

```diff
@@ -16,7 +16,7 @@ def get(ports, settings, shared):
 
   # get the port
   ports.fetch_project('sdl2', 'https://github.com/emscripten-ports/SDL2/archive/' + TAG + '.zip', SUBDIR)
-  libname = ports.get_lib_name('libSDL2')
+  libname = ports.get_lib_name('libSDL2' + ('-mt' if settings.USE_PTHREADS else ''))
```

Comment:
This needs integration in embuilder.py, so that we can build the `-mt` version there too.